shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -88,8 +113,10 @@ class FlutterPlatformViewsController {
   std::map<std::string, fml::scoped_nsobject<NSObject<FlutterPlatformViewFactory>>> factories_;
   std::map<int64_t, fml::scoped_nsobject<NSObject<FlutterPlatformView>>> views_;
   std::map<int64_t, fml::scoped_nsobject<FlutterTouchInterceptingView>> touch_interceptors_;
+  std::map<int64_t, fml::scoped_nsobject<UIView>> root_views_;
```

Comment:
Can you add a comment explaining this and clip_count_?

Reply:
Done